STM32G070RBTx code won't linking when use lowpower shutdown

```
#include "STM32LowPower.h"

void setup() {
  // Configure low power
  LowPower.begin();
  Serial.begin(115200);
  Serial.println("\r\nstarted");
  Serial.flush();
}

void loop() {
  Serial.println("idle");
  Serial.flush();
  LowPower.idle(5000);

  Serial.println("sleep");
  Serial.flush();
  LowPower.sleep(5000);

  Serial.println("deep sleep");
  Serial.flush();
  LowPower.deepSleep(5000);

  Serial.println("shutdown");
  Serial.flush();
  LowPower.shutdown(5000);
}
```

```
Compiling core...
/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/10.3.1-2.3/bin/arm-none-eabi-gcc -mcpu=cortex-m0plus -DUSE_FULL_LL_DRIVER -mthumb @/private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/sketch/build.opt -c -Os -DNDEBUG -Wall -Wextra -std=gnu11 -ffunction-sections -fdata-sections --param max-inline-insns-single=500 -MMD -I/Users/william/Documents/Arduino/stm32f0_test -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/avr -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/LL -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/hid -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/cdc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/STM32G0xx -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/open-amp/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/libmetal/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/virtual_driver -DSTM32G0xx -DARDUINO=10607 -DARDUINO_GENERIC_G070RBTX -DARDUINO_ARCH_STM32 "-DBOARD_NAME=\"GENERIC_G070RBTX\"" "-DVARIANT_H=\"variant_generic.h\"" -DSTM32G070xx -DHAL_UART_MODULE_ENABLED -D__CORTEX_SC=0 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/Core/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Source/Templates/gcc/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/Include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/PrivateInclude -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT /Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT/PeripheralPins.c -o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/PeripheralPins.c.o
/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/10.3.1-2.3/bin/arm-none-eabi-g++ -mcpu=cortex-m0plus -DUSE_FULL_LL_DRIVER -mthumb @/private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/sketch/build.opt -c -Os -DNDEBUG -Wall -Wextra -std=gnu++14 -ffunction-sections -fdata-sections -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -fno-use-cxa-atexit -MMD -I/Users/william/Documents/Arduino/stm32f0_test -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/avr -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/LL -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/hid -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/cdc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/STM32G0xx -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/open-amp/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/libmetal/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/virtual_driver -DSTM32G0xx -DARDUINO=10607 -DARDUINO_GENERIC_G070RBTX -DARDUINO_ARCH_STM32 "-DBOARD_NAME=\"GENERIC_G070RBTX\"" "-DVARIANT_H=\"variant_generic.h\"" -DSTM32G070xx -DHAL_UART_MODULE_ENABLED -D__CORTEX_SC=0 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/Core/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Source/Templates/gcc/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/Include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/PrivateInclude -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT /Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT/variant_generic.cpp -o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/variant_generic.cpp.o
/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/10.3.1-2.3/bin/arm-none-eabi-gcc -mcpu=cortex-m0plus -DUSE_FULL_LL_DRIVER -mthumb @/private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/sketch/build.opt -c -Os -DNDEBUG -Wall -Wextra -std=gnu11 -ffunction-sections -fdata-sections --param max-inline-insns-single=500 -MMD -I/Users/william/Documents/Arduino/stm32f0_test -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/avr -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/LL -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/hid -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/cdc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/STM32G0xx -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/open-amp/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/libmetal/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/virtual_driver -DSTM32G0xx -DARDUINO=10607 -DARDUINO_GENERIC_G070RBTX -DARDUINO_ARCH_STM32 "-DBOARD_NAME=\"GENERIC_G070RBTX\"" "-DVARIANT_H=\"variant_generic.h\"" -DSTM32G070xx -DHAL_UART_MODULE_ENABLED -D__CORTEX_SC=0 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/Core/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Source/Templates/gcc/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/Include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/PrivateInclude -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT /Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT/generic_clock.c -o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/generic_clock.c.o
/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/10.3.1-2.3/bin/arm-none-eabi-g++ -mcpu=cortex-m0plus -DUSE_FULL_LL_DRIVER -mthumb @/private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/sketch/build.opt -c -Os -DNDEBUG -Wall -Wextra -std=gnu++14 -ffunction-sections -fdata-sections -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -fno-use-cxa-atexit -MMD -I/Users/william/Documents/Arduino/stm32f0_test -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/avr -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/LL -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/hid -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino/stm32/usb/cdc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/STM32G0xx_HAL_Driver/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/STM32G0xx -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Inc -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/ST/STM32_USB_Device_Library/Core/Src -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/open-amp/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/libmetal/lib/include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Middlewares/OpenAMP/virtual_driver -DSTM32G0xx -DARDUINO=10607 -DARDUINO_GENERIC_G070RBTX -DARDUINO_ARCH_STM32 "-DBOARD_NAME=\"GENERIC_G070RBTX\"" "-DVARIANT_H=\"variant_generic.h\"" -DSTM32G070xx -DHAL_UART_MODULE_ENABLED -D__CORTEX_SC=0 -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/Core/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Include/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/Drivers/CMSIS/Device/ST/STM32G0xx/Source/Templates/gcc/ -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/Include -I/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/PrivateInclude -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/cores/arduino -I/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT /Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT/variant_NUCLEO_G070RB.cpp -o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/variant_NUCLEO_G070RB.cpp.o
Using precompiled core: /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-core-cache/core_b58712e53bbb325a5468e76db2e37825.a
Linking everything together...
/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/10.3.1-2.3/bin/arm-none-eabi-gcc -mcpu=cortex-m0plus -mthumb -Os -DNDEBUG --specs=nano.specs -Wl,--defsym=LD_FLASH_OFFSET=0 -Wl,--defsym=LD_MAX_SIZE=131072 -Wl,--defsym=LD_MAX_DATA_SIZE=36864 -Wl,--cref -Wl,--check-sections -Wl,--gc-sections -Wl,--entry=Reset_Handler -Wl,--unresolved-symbols=report-all -Wl,--warn-common -Wl,--default-script=/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/variants/STM32G0xx/G070RBT/ldscript.ld -Wl,--script=/Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/system/ldscript.ld -Wl,-Map,/private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/stm32f0_test.ino.map -L/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/CMSIS/5.7.0/CMSIS/DSP/Lib/GCC/ -larm_cortexM0l_math -o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/stm32f0_test.ino.elf -L/private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8 -Wl,--start-group /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/sketch/SrcWrapper.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/sketch/stm32f0_test.ino.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/STM32duino_Low_Power/STM32LowPower.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/STM32duino_Low_Power/low_power.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/STM32duino_RTC/STM32RTC.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/STM32duino_RTC/rtc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_adc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_adc_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_can.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_cec.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_comp.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_comp_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_cordic.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_cortex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_crc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_crc_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_cryp.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_cryp_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dac.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dac_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dcache.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dcmi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dcmi_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dfsdm.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dfsdm_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dma.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dma2d.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dma_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dsi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_dts.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_eth.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_eth_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_exti.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_fdcan.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_firewall.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_flash.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_flash_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_flash_ramfunc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_fmac.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_fmpi2c.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_fmpi2c_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_fmpsmbus.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_fmpsmbus_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_gfxmmu.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_gpio.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_gpio_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_gpu2d.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_gtzc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_hash.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_hash_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_hcd.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_hrtim.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_hsem.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_i2c.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_i2c_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_i2s.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_i2s_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_icache.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_ipcc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_irda.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_iwdg.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_jpeg.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_lcd.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_lptim.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_ltdc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_ltdc_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_mdf.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_mdios.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_mdma.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_mmc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_mmc_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_nand.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_nor.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_opamp.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_opamp_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_ospi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_otfdec.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pccard.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pcd.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pcd_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pka.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pssi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pwr.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_pwr_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_qspi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_ramcfg.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_ramecc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_rcc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_rcc_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_rng.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_rng_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_rtc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_rtc_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sai.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sai_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sd.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sd_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sdadc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sdram.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_smartcard.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_smartcard_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_smbus.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_smbus_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_spdifrx.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_spi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_spi_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_sram.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_subghz.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_swpmi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_tim.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_tim_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_tsc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_uart.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_uart_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_usart.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_usart_ex.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_wwdg.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HAL/stm32yyxx_hal_xspi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/HardwareTimer.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_adc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_bdma.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_comp.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_cordic.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_crc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_crs.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_dac.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_delayblock.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_dlyb.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_dma.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_dma2d.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_exti.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_fmac.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_fmc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_fmpi2c.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_fsmc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_gpio.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_hrtim.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_i2c.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_icache.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_lpgpio.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_lptim.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_lpuart.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_mdma.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_opamp.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_pka.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_pwr.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_rcc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_rng.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_rtc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_sdmmc.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_spi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_swpmi.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_tim.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_ucpd.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_usart.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_usb.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/LL/stm32yyxx_ll_utils.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/new.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/PortNames.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/analog.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/bootloader.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/clock.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/core_callback.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/dwt.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/hw_config.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/interrupt.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/otp.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/pinmap.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/stm32_def.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/system_stm32yyxx.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/timer.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/stm32/uart.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/SrcWrapper/syscalls.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/PeripheralPins.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/generic_clock.c.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/variant_NUCLEO_G070RB.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/core/variant_generic.cpp.o /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-core-cache/core_b58712e53bbb325a5468e76db2e37825.a -lc -Wl,--end-group -lm -lgcc -lstdc++
/Users/william/Library/Arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/10.3.1-2.3/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /private/var/folders/qt/0_9tcc1s0w33_stsb1h9yhvc0000gn/T/arduino-sketch-6717264A6CEB5902D8DFBCCD107C59A8/libraries/STM32duino_Low_Power/low_power.c.o: in function `LowPower_shutdown':
low_power.c:(.text.LowPower_shutdown+0x12): undefined reference to `HAL_PWREx_EnterSHUTDOWNMode'
collect2: error: ld returned 1 exit status

Using library STM32duino_Low_Power at version 1.2.1 in folder: /Users/william/Documents/Arduino/libraries/STM32duino_Low_Power 
Using library STM32duino_RTC at version 1.3.2 in folder: /Users/william/Documents/Arduino/libraries/STM32duino_RTC 
Using library SrcWrapper at version 1.0.1 in folder: /Users/william/Library/Arduino15/packages/STMicroelectronics/hardware/stm32/2.3.0/libraries/SrcWrapper 
exit status 1

Compilation error: exit status 1
```

--- a/src/low_power.c
+++ b/src/low_power.c
@@ -548,7 +548,7 @@ void LowPower_shutdown()
   __HAL_PWR_CLEAR_FLAG(PWR_MPUCR_CSSF);
 #endif
 
-#if defined(PWR_CR1_LPMS)
+#if defined(PWR_LOWPOWERMODE_SHUTDOWN) || defined(PWR_CR1_LPMS_SHUTDOWN) || defined(LL_PWR_SHUTDOWN_MODE)
   /* LSE must be on to use shutdown mode */
   if (__HAL_RCC_GET_FLAG(RCC_FLAG_LSERDY) == SET) {
     HAL_PWREx_EnterSHUTDOWNMode();

